### ISSUE TITLE
Construct plantuml URL problem

### ISSUE THREAD

**Issue by sdash-github** [PeanutBase/jekyll-peanutbase#44]
File:
@dev-pb-jekyll /var/www/jekyll-peanutbase/userguides/jb2/jb2_genomeStrCompare.html
This is the img tag src I need:
```<img src="http://www.plantuml.com/plantuml/proxy?cache=no&src=https://dev.pb.lis.ncgr.org/userguides/jb2/jb2_genomeStrCompare-act.puml">  ```
This works in "https://dev.pb.lis.ncgr.org" but obviously not in 'dev.peanutbase.org' nor in 'peanutbase.org'. So, I have been testing using hardcoded server URL for each server.

My attempt:
```
{% assign url_plantumlImage = "http://www.plantuml.com/plantuml/proxy?cache=no&src=https://" | append: "{{ site.devsite }}" | append: ".org/userguides/jb2/jb2_genomeStrCompare-act.puml"  %}    
...  ...
<img src="{{ url_plantumlImage }}">   
<p> host: "{{ site.devsite }}" </p>
```
**site.devsite is defined in _config.yml in devsite to get around for testing as site.url variable didn't yield the correct URL.

Definitely need help: Has spent many hours looking for help in jekyll liquid template and didn't succeed. Assigning Sven to help me out here and want @adf to watch if he can give some pointers.

**Comment by svengato** [comment]
I cannot see either site from the internal network:
dev.peanutbase.org: Did Not Connect: Potential Security Issue
[www.]peanutbase.org: times out

**Comment by svengato** [comment]
I can see dev.pb.lis.ncgr.org, trying to understand that.

**Comment by sdash-github** [comment]
Observation:
After @adf-ncgr fixed issue #43,  src="https://peanutbase.org ..." works in dev.peanutbase.org site. For some reason it is recognized as same domain.  
Should close the issue but should check a few more times before that. All is well that ends well!

**Comment by sdash-github** [comment]
Closing issue as no need now for getting site.url or site hostname as cross domain problem doesn't exist.
